precompile/util/const.go

```diff
@@ -0,0 +1,11 @@
+package util
+
+import (
+	"math/big"
+)
+
+// Coin for converting sdk.Coin Amount from sdkmath.Int to *big.Int
+type Coin struct {
```

Comment:
I think it's better replace our Struct.sol including the Coin struct, which includes Types.sol and type.go from cosmos/evm. This will avoid problems caused by other types in the future.

Reply:
[e92f16e](https://github.com/xpladev/xpla/pull/176/commits/e92f16e49d9fa66bdd9b60eb857f3cf18f871b03)

Reply:
+ It seems that the util functions `GetCoin` and `GetCoins` that we created can also be replaced with `ToCoins`.

Reply:
@yoosah